cookbook/deployment/heroku.rst

```diff
@@ -106,6 +106,15 @@ directory of the application and add just the following content:
 
     web: bin/heroku-php-apache2 web/
 
+.. note::
+
+    Nginx is also available on Heroku. If you prefer to use it, you can create a
+    Nginx config file (``nginx_app.conf`` for instance) to point from Procfile as described in `Heroku docs`_:
```

Comment:
I think we should reword the paragraph a bit like this:

> If you prefer to use Nginx, which is also available on Heroku, you can create a configuration file for it and point to it from your Procfile as described in the Heroku documentation.

What do you think @phansys?